### ISSUE TITLE
Crashing 

### ISSUE THREAD

**Issue by pepek1122583** [BiomeBundle/BiomeBundle#107]
hi, I dont why but server start crashing on several chunks
[crash-2017-12-04_21.15.31-server.txt](https://github.com/BiomeBundle/BiomeBundle/files/1528650/crash-2017-12-04_21.15.31-server.txt)

**Comment by MCPitman** [comment]
This has not been reported by anyone else which is surprising if it's an actual issue. 

What version of Biome Bundle are you using? Have you edited something in biome bundle like adding new objects?

**Comment by sometimesopensgm** [comment]
i'm also getting a crash, albeit with multiple mods
seems to be linked to biome bundle
using latest 1.12 version, seems to be linked to savannahs
[crashlog here](https://github.com/BiomeBundle/BiomeBundle/files/1801015/big.crash.txt)

**Comment by ispublic** [comment]
I experience a similar (the same?) error with a lightweight BB 1.22.2 v6 modpack.

After some finagling, I isolated the region in which the error is taking place and deleted it, allowing me to log in and teleport to and away. Gameplay is normal outside of the affected region. Any player in the affected region causes a server crash.

Once the first exception is thrown, the server will throw the following error [ `[Server thread/INFO] [OTG]: SaveToDisk waiting on Populate. Although other mods could be causing this and there may not be any problem, this can potentially cause an endless loop!` ]for a few hundred thousand lines, so I am not posting the server log itself. Two crash logs are attached, the initial crashlog and the one from when I returned to the region and allowed it to regenerate.
[crash-2018-12-30_22.32.00-server.txt](https://github.com/BiomeBundle/BiomeBundle/files/2717577/crash-2018-12-30_22.32.00-server.txt)
[crash-2018-12-30_21.09.44-server.txt](https://github.com/BiomeBundle/BiomeBundle/files/2717578/crash-2018-12-30_21.09.44-server.txt)
